app/chain_api.py

```diff
@@ -43,7 +43,10 @@ def __init__(self, services):
                 core_agent=lambda d: self.data_svc.explode_agents(criteria=d),
                 core_group=lambda d: self.data_svc.explode_groups(criteria=d),
                 core_result=lambda d: self.data_svc.explode_results(criteria=d),
-            )
+            ),
+            DELETE=dict(
+                core_group=lambda d: self.data_svc.deactivate_group(**d),
```

Comment:
when you make this change in the future (linked to the change in the core code) make sure you "link" the two MRs.